api/pkg/transformer/pipeline/compiler.go

```diff
@@ -198,10 +214,32 @@ func (c *Compiler) parseTablesSpec(tableSpecs []*spec.Table, compiledJsonPaths *
 	return NewCreateTableOp(tableSpecs), nil
 }
 
+func (c *Compiler) parseTableTransform(transformationSpecs *spec.TableTransformation, paths *jsonpath.Storage, compiledExpressions *expression.Storage) (Op, error) {
+	for _, step := range transformationSpecs.Steps {
+		for _, updateColumn := range step.UpdateColumns {
+			compiledExpression, err := c.compileExpression(updateColumn.Expression)
+			if err != nil {
+				return nil, err
+			}
+			compiledExpressions.Set(updateColumn.Expression, compiledExpression)
+		}
+	}
+
```

Comment:
i think you need to register dummy variables for outputTable

Reply:
also can you add test case when using array of transformation when the second transformation using output of previous transformation?

Reply:
Updated